ts/src/timex.ts

```diff
@@ -684,10 +684,10 @@ export default class timex extends Exchange {
         }
         if (since !== undefined) {
             request['from'] = this.iso8601 (since);
-            request['till'] = this.iso8601 (this.sum (since, this.sum (limit, 1) * duration * 1000));
```

Comment:
this is a bug, you need to keep the till here since it is in the request and not the params

Reply:
Hi frosty